### PR TITLE
Traffic analyzer standalone entry

### PR DESCRIPTION
**Pull request by didovesei**

### CODE REVIEW

File: daq/utils.py [hunk 1]
```diff
@@ -34,3 +38,11 @@ def proto_json(message):
 def dict_proto(message, proto_func, ignore_unknown_fields=False):
     """Convert a standard dict object to a proto object"""
     return json_format.ParseDict(message, proto_func(), ignore_unknown_fields)
+
+
+def write_pid_file(pid_file, logger=None):
+    pid = os.getpid()
+    if logger:
+        logger.info('pid is %d', pid)
```
Comment (grafnu) [inline daq/utils.py:46]: Maybe add path of PID file?  Either way...

File: proto/acl_counting.proto [hunk 2]
```diff
@@ -4,15 +4,16 @@
 syntax = "proto3";
 
 /*
- * ACL counts for devices
+ * Rule counts of devices
```
Comment (grafnu) [inline proto/acl_counting.proto:7]: nit: Rule counts *for* devices.  "of" implies that the keys are Rules and the counts of number of devices.

File: proto/acl_counting.proto [hunk 3]
```diff
@@ -4,15 +4,16 @@
...
  */
-message AclCounts {
-  // List of ACL counts indexed by device MAC addresses
-  map<string, RuleCounts> devices = 1;
+message DeviceRuleCounts {
+  // List of Rule counts indexed by device MAC addresses
+  map<string, RuleCounts> device_mac_rules = 1;
+  string error = 2;
```
Comment (grafnu) [inline proto/acl_counting.proto:12]: comment for field

File: proto/acl_counting.proto [hunk 4]
```diff
@@ -4,15 +4,16 @@
...
 }
 
 /*
- *  ACL counts for a device
+ *  Rule counts for a device
```
Comment (grafnu) [inline proto/acl_counting.proto:16]: This one seems correct.

File: daq/utils.py (hunk 1 above)
Comment (didovesei) [reply daq/utils.py]: Added file path in the log message

File: proto/acl_counting.proto (hunk 2 above)
Comment (didovesei) [reply proto/acl_counting.proto]: Done

File: proto/acl_counting.proto (hunk 3 above)
Comment (didovesei) [reply proto/acl_counting.proto:13]: Done

**Comment by didovesei** [comment]
Resolved the comments and completed the integration test. PTAL..

File: proto/acl_counting.proto [hunk 5]
```diff
@@ -4,11 +4,12 @@
 syntax = "proto3";
 
 /*
- * Rule counts of devices
+ * Rule counts for devices
  */
 message DeviceRuleCounts {
   // List of Rule counts indexed by device MAC addresses
   map<string, RuleCounts> device_mac_rules = 1;
+  // error found when getting the rule counts for the devices
```
Comment (grafnu) [inline proto/acl_counting.proto:12]: nit; space between entries

File: proto/acl_counting.proto [hunk 6]
```diff
@@ -18,6 +19,7 @@ message DeviceRuleCounts {
 message RuleCounts {
   // List of rule counts indexed by the descriptions of the rules
   map<string, RuleCount> rules = 1;
+  // errors found when analyzing the rule counts of a device
```
Comment (grafnu) [inline proto/acl_counting.proto:22]: nit: space between entries

File: testing/test_mud.sh [hunk 7]
```diff
@@ -22,7 +22,8 @@ cat inst/run-9a02571e8f00/nodes/nmap01/activate.log
 
 echo %%%%%%%%%%%%%%%%%%%%%% Mud profile tests | tee -a $TEST_RESULTS
 rm -f local/system.yaml
-echo 'include=../config/system/muddy.conf' > local/system.conf
+echo 'include=../config/system/muddy.conf
```
Comment (grafnu) [inline testing/test_mud.sh:25]: for multiline use <<EOF syntax, see daq/testing/test_many.sh for example

Comment (didovesei) [reply testing/test_mud.sh]: Updated. PTAL..

**Review by grafnu — APPROVED** [review]

**Pull request merged**